File: 0392-is-subsequence/0392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        
        int j = 0, k = s.size();
        for(int i=0; j < k && i<t.size();i++){
            
            if(s[j] == t[i]){
                j++;
            }        
        }
        if(k == j){
            return true;
        }
            return false;        
    }
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        slow = 0
        s_len = len(s)
    
        for fast in range(len(t)):
            if slow < s_len and s[slow] == t[fast]:
                slow += 1
        if slow == s_len:
            return True
        return False